community/kernel/src/test/java/org/neo4j/graphdb/IdReuseTest.java

```diff
@@ -65,15 +73,15 @@ public void shouldReuseRelationshipIdsFromRolledBackTransaction() throws Excepti
         // Given
         GraphDatabaseService db = dbRule.getGraphDatabaseService();
         Node node1, node2;
-        try (Transaction tx = db.beginTx())
```

Comment:
This is all good and fixing checkstyle anomalies, but to do it in maintenance branches causes more harm than good usually.